5.x/crud-operation-list-entries.md

```diff
@@ -143,6 +143,35 @@ Using `'visibleInTable' => false` will make the field start hidden in the table.
 
 If you want a field that is not on table, user can't show it, but will **ALWAYS** be exported use the `exportOnlyField => true`. If used will ignore any other custom visibility you defined.
 
+#### Custom dataTableConfiguration
+
+<a name="custom-dataTableConfiguration"></a>
+
+Example changing field separator in dataTable export to use semicolon (;) instead of comma (,) :
+
+copy vendor/backpack/crud/src/resources/views/crud/inc/export_buttons.blade.php to resources/views/vendor/backpack/crud/inc/export_buttons.blade.php
+
+```php
+{
+    name: 'csvHtml5',
+    extend: 'csvHtml5',
+    fieldSeparator: ';',
+    exportOptions: {
+        columns: function ( idx, data, node ) {
+            var $column = crud.table.column( idx );
+                return  ($column.visible() && $(node).attr('data-visible-in-export') != 'false') || $(node).attr('data-force-export') == 'true';
+        }
+    },
+    action: function(e, dt, button, config) {
+        crud.responsiveToggle(dt);
+        $.fn.DataTable.ext.buttons.csvHtml5.action.call(this, e, dt, button, config);
+        crud.responsiveToggle(dt);
+    }
+},
+```
```

Comment:
```suggestion
#### How to use different separator in DataTables (eg. semicolon instead of comma)

<a name="custom-dataTableConfiguration"></a>

If you want to change the separator in dataTable export to use semicolon (;) instead of comma (,) :

**Step 1.** Copy vendor/backpack/crud/src/resources/views/crud/inc/export_buttons.blade.php to resources/views/vendor/backpack/crud/inc/export_buttons.blade.php

**Step 2.** Change it in your `dataTableConfiguration`:
```php
{
    name: 'csvHtml5',
    extend: 'csvHtml5',
    fieldSeparator: ';',
    exportOptions: {
        columns: function ( idx, data, node ) {
            var $column = crud.table.column( idx );
                return  ($column.visible() && $(node).attr('data-visible-in-export') != 'false') || $(node).attr('data-force-export') == 'true';
        }
    },
    action: function(e, dt, button, config) {
        crud.responsiveToggle(dt);
        $.fn.DataTable.ext.buttons.csvHtml5.action.call(this, e, dt, button, config);
        crud.responsiveToggle(dt);
    }
},
```
```